packages/compiled-assets/package.json

```diff
@@ -16,7 +16,7 @@
   "dependencies": {
     "@prairielearn/html": "workspace:^",
     "commander": "^11.0.0",
-    "esbuild": "^0.18.17",
+    "esbuild": "^0.18.7",
```

Comment:
Can you clarify why this was downgraded below the previous version?

Reply:
This was a typo when I was manually downgrading `esbuild`, which I did to avoid having two copies installed. Once `tsup` is changed to use `^0.19.0`, we can upgrade everything to that new version range.